src/ui/public/vislib/lib/axis.js

```diff
@@ -68,7 +70,7 @@ export default function AxisFactory(Private) {
     }
 
     getLength(el, n) {
-      const margin = this.config.get('vis._attr.margin');
+      const margin = this.config._chartConfig.get('style.margin');
```

Comment:
Interesting, I look forward to seeing how you handle this. No need to fix it here.